simpleocr/pillow_utils.py

```diff
@@ -0,0 +1,27 @@
+from .files import Image
+from PIL import Image
+import numpy
+import cv2
+
+
+def imagebuffer_to_pil(imagefile):
```

Comment:
Sorry, I forgot to rename these functions (it goes for the `from_file` tests as well).